### PR TITLE
Update install instructions and some minor fixes

### PR DESCRIPTION
**Pull request by henri-hulski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,13 +21,15 @@ A state controller with its own debugger
 ## Try out Cerebral 2
 To try out Cerebral 2 you can install any package with:
 
-`npm install {cerebralPackage}@next --save --save-exact`
+`npm install {cerebralPackage}@beta --save`
+
+For packages which are scoped with `@cerebral/` `@beta` can be omitted.
```
Comment (Guria) [inline README.md]: It is just a side effect of first publish. Lets omit this note.

File: README.md [hunk 2]
```diff
@@ -21,13 +21,15 @@ A state controller with its own debugger
...
 
 This will install the latest version of that package. New versions are automatically deployed whenever pull requests are merged into master.
 
 When you want to update your packages you **have to update all used packages in one go**, for example:
```
Comment (Guria) [inline README.md]: Hope to get rid of this and have separate versioning soon.

File: packages/firebase-admin/README.md [hunk 3]
```diff
@@ -3,11 +3,11 @@
 ## install
 **NPM**
 
-`npm install @cerebral/firebase-admin@next --save --save-exact`
+`npm install @cerebral/firebase-admin --save`
```
Comment (Guria) [inline packages/firebase-admin/README.md]: Lets keep beta tag here and for other packages

File: README.md (hunk 1 above)
Comment (henri-hulski) [inline README.md]: @Guria Why you want to keep beta tag? There are no Cerebral 1 packages with this name and after release the beta tag will be removed.

File: README.md (hunk 2 above)
Comment (henri-hulski) [inline README.md]: So remove this now?

Comment (Guria) [inline README.md]: lets keep it until we figure out how we deal with it.

File: README.md (hunk 1 above)
Comment (Guria) [inline README.md]: just to be on safe side before release. eg router will be updated and potentially could break smth if updated automatically.

**Pull request merged**